<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PcapSplit_GUIClass</class>
 <widget class="QMainWindow" name="PcapSplit_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PcapSplit_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start_Btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop_Btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Save_Text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Time_Text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Port_Text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>175</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>75</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DeviceN.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DeviceNo_comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="PBar_PS">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="show_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PcapSplit_GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Start_Btn</sender>
   <signal>clicked()</signal>
   <receiver>PcapSplit_GUIClass</receiver>
   <slot>Btn_Start_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Stop_Btn</sender>
   <signal>clicked()</signal>
   <receiver>PcapSplit_GUIClass</receiver>
   <slot>Btn_Stop_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Save_Btn</sender>
   <signal>clicked()</signal>
   <receiver>PcapSplit_GUIClass</receiver>
   <slot>Btn_Save_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeviceNo_comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>PcapSplit_GUIClass</receiver>
   <slot>Combox_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Btn_Start_Click()</slot>
  <slot>Btn_Stop_Click()</slot>
  <slot>Btn_Save_Click()</slot>
  <slot>Combox_Click()</slot>
 </slots>
</ui>
